recipes/r-cpprouting/meta.yaml

```
@@ -0,0 +1,110 @@
+{% set version = '3.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cpprouting
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/cppRouting_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/cppRouting/cppRouting_{{ version }}.tar.gz
+  sha256: bed3a5545f4e8b2ad849735c499149a5ee668d032ec54eb9ce1e523ec53417d6
+
+build:
+  skip: True  # [win]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    
+    - lib/
+    - lib/R/library/RcppParallel/libs/  # [osx]
+    
+
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+    - '*/libtbb.dylib'   # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp >=1.0.7
+    - r-rcppparallel
+    - r-rcppprogress
+    - r-data.table
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp >=1.0.7
+    - r-rcppparallel
+    - r-rcppprogress
+    - r-data.table
+
+test:
+  commands:
+    - $R -e "library(RcppParallel); library(cppRouting); RcppParallel::setThreadOptions(numThreads = 2); edges<-data.frame(from_vertex=c(0,0,1,1,2,2,3,4,4), to_vertex=c(1,3,2,4,4,5,1,3,5), cost=c(9,2,11,3,5,12,4,1,6), dist = c(5,3,4,7,5,5,5,8,7)); graph <- cppRouting::makegraph(edges[,1:3], directed=TRUE, aux = edges\$dist); nodes <- graph\$dict\$ref; time_mat <- cppRouting::get_distance_pair(graph, from = nodes, to = nodes); dist_mat <- cppRouting::get_distance_pair(graph, from = nodes, to = nodes, aggregate_aux = TRUE); stopifnot(all(time_mat == c(0,0,0,0,0,0))); stopifnot(all(dist_mat == c(0,0,0,0,0,0)))"  # [not win]
+    #- "\"%R%\" -e \"library(RcppParallel)"  # [win]
```

Would you minding rolling this back to a test file? Also, we usually like to retain the loading tests separately and keep symmetric branches for win even if they aren't visited. E.g., something like:

```suggestion
    - $R -e "library(cppRouting)"  # [not win]
    - $R -f test.R  # [not win]
    - "\"%R%\" -e \"library(cppRouting)"  # [win]
    - "\"%R%\" -f test.R"  # [win]
```